poms/bom-test/pom.xml

```diff
@@ -54,6 +54,13 @@
                 <type>pom</type>
                 <scope>import</scope>
             </dependency>
+            <dependency>
+                <groupId>io.quarkiverse.artemis</groupId>
+                <artifactId>quarkus-artemis-bom</artifactId>
```

Comment:
I was not able to find this in a maven repo. Could you please provide a link to have a look ?

Reply:
Oh wait, why did worked for me ?

Reply:
I did not try to build @zbendhiba . I was just looking for the pom on sites like https://mvnrepository.com/search?q=quarkus-artemis-bom.

Reply:
https://mvnrepository.com is a bit delayed. It is there in the Central https://repo1.maven.org/maven2/io/quarkiverse/artemis/quarkus-artemis-bom/1.0.2/quarkus-artemis-bom-1.0.2.pom